exonum/src/node/mod.rs

```diff
@@ -752,7 +752,7 @@ impl ApiSender {
 impl TransactionSend for ApiSender {
     fn send(&self, tx: Box<dyn Transaction>) -> Result<(), failure::Error> {
         if !tx.verify() {
-            bail!("Unable to verify transaction");
+            bail!("Invalid transaction");
```

Comment:
Let me ask Why ? 😄

Reply:
I think I got the idea: we are actually able to the verify transaction, but because this method returns `false` - the transaction is considered invalid. But I would expand description a little bit. For example, "Invalid transaction (verify() returns false)".